percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -139,6 +134,15 @@ get_sources(){
         git clean -xdf
         git checkout "$BRANCH"
     fi
+
+    if [ -z "${PSM_VER}" ]; then
+        PSM_VER=$(git describe --tags --abbrev=0 | sed 's/^psmdb-//' | awk -F '-' '{if ($2 ~ /^rc/) {print $0} else {print $1}}'
+    fi
+
+    if [ -z "${PSM_RELEASE}" ]; then
+        PSM_RELEASE=$(git describe --tags --abbrev=0 | sed 's/^psmdb-//' | awk -F '-' '{if ($2 ~ /^rc/) {print 1} else {print $2}}'
```

Comment:
There is a typo: no $ sign in {print 1}

Reply:
It is not a typo, If we do not have a release we set '1' for PSM_RELEASE.